docs/release-notes.md

```diff
@@ -16,6 +16,14 @@ Packaging changes:
 
 
 
+## Afterburn 5.4.3 (2023-06-05)
+
+Minor changes:
+
+- Cargo update all dependencies to avoid missing data in vendor tarball
+
```

Comment:
One too many trailing newlines.  There should be two between releases.

Reply:
Ugh, sorry I followed the pattern from the (unreleased) section. I should have known better, and double checked.